### PR TITLE
Feature/ge 1098

### PR DESCRIPTION
**Pull request by AltusBarry**
Added new locales

Added middleware subclassing Django LocalMiddleware, that will attempt to redirect to the language requested via querystring or 404 as intended.

### CODE REVIEW

File: authentication_service/middleware.py [hunk 1]
```diff
@@ -224,3 +231,59 @@ class ErrorMiddleware(MiddlewareMixin):
...
+
+    def process_request(self, request):
+        super(GELocaleMiddleware, self).process_request(request)
+
+        # Get the language code to use
+        language_code = request.GET.get(
+            LANGUAGE_QUERY_PARAMETER, None
+        )
+
+
```
Comment (cobusc) [inline authentication_service/middleware.py]: Unnecessary whitespace

File: authentication_service/middleware.py [hunk 2]
```diff
@@ -224,3 +231,59 @@ class ErrorMiddleware(MiddlewareMixin):
...
+    Overrides the default logic to ALWAYS attempt to switch to the language
+    provided via querystring.
+
+    Most of this code was inspired by
+    django.middleware.locales.LocaleMiddleware itself.
+    """
+
+    def process_request(self, request):
+        super(GELocaleMiddleware, self).process_request(request)
```
Comment (cobusc) [inline authentication_service/middleware.py]: Why does it feel like this should either be at the end of this function, i.e. only called if no redirect is done?

**Comment by cobusc** [comment]
:+1: Nice work!

File: authentication_service/middleware.py (hunk 2 above)
Comment (AltusBarry) [reply authentication_service/middleware.py]: The super also activates a language. So if we are already on the correct language we need not redirect.
